### ISSUE TITLE
Related assets component

### ISSUE THREAD

**Issue by godanny86** [Adobe-Marketing-Cloud/asset-share-commons#82]
It would be nice to have a "Related Assets" component that could be dropped on to the Asset Details page. The Related Assets component would display a list of related assets. 

* configurable max number of assets to display
* could simply output the values of the [ootb Assets Feature](https://helpx.adobe.com/experience-manager/6-3/assets/using/related-assets.html)
* or related assets based on a query of assets with the same tags
* or related assets based on sibling assets in the same folder

cache considerations would need to be taken into account

**Comment by davidjgonzalez** [comment]
I an interesting consideration here is:
Assuming these related assets will appear on the Asset Details page (in some list)... should the "base" Asset Share's query restrictions (path, expired assets, etc.) be applied to this as well? For example, if the asset share is configured to limit results to /content/dam/foo .. .and related finds a match under /content/dam/bar ... should it display that result? (or expired or main assets)?

If so, how do we restrict?

And lastly, what is the navigation pattern for this...where will the Back-button in the Details title component go? IIRC current impl takes them back to the search page, vs what the browser things of as "back" (the prior asset details page)

**Comment by godanny86** [comment]
@schmidtkevina, I understand your current implementation is looking for this. If you want to contribute back to the project that'd be great.

**Comment by schmidtkevina** [comment]
Yes, once we have our requirements finalized I will share here and will contribute when finished.
